app/helpers/footer_helper.rb

```diff
@@ -2,10 +2,10 @@ module FooterHelper
   def footer_meta_items
     [
       { text: t(".guidance"), href: "#" },
-      { text: t(".accessibility"), href: "#" },
-      { text: t(".cookies"), href: "#" },
+      { text: t(".accessibility"), href: claims_accessibility_path },
```

Comment:
This isn't going to work, because Placements also use this `footer_helper` with the `FooterComponent` in the `layouts/_footer.html.erb`
Could you make this service conditional.

Reply:
Good catch